analyzers/nodejs/integration_test.go

```diff
@@ -0,0 +1,210 @@
+package nodejs_test
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	testUtils "github.com/fossas/fossa-cli/testing"
```

Comment:
This naming is kind of annoying, given the clash with stdlib's `testing`. What are your thoughts on using subpackages? My initial thinking:
- `testing/fixtures` (for fixture functions: consumers would call e.g. `fixtures.Clone(...)`)
- `testing/runfossa` (for invoking `fossa` commands end-to-end e.g. `runfossa.Init(...)`).

Reply:
I think that seems reasonable